.github/workflows/label_issues.yml

```diff
@@ -0,0 +1,21 @@
+name: Label issues
+on:
+  issues:
+    types:
+      - reopened
```

Comment:
should this work for reopened as well? I imagine we could reopen some accidently closed feature issue or something

Reply:
I guess uses won't be reopening issues by themselves often. I'll remove it.